### ISSUE TITLE
wrong directory for observr in Readme

### ISSUE THREAD

**Issue by bo-tato** [edgecase/ruby_koans#46]
In the readme it says:
```
cd ruby_koans
rake
# decide to run rake automatically from now on as you edit
gem install observr
observr ./koans/koans.watchr
```

Running it like that outside the koans directory will get a "No rakefile found" error, it should be run in the koans directory just as `observr koans.watchr`

**Comment by tonywok** [comment]
Would you might making a patch? I’ve never come across observr 😅

While I don’t have any real strong inclination, my initial instinct is to maybe even remove the mention of it all together as this targets primarily folks learning ruby. What do you think?

**Comment by bo-tato** [comment]
sure I submitted a PR, it's a really small change. If you want to remove it that'd be reasonable to in the spirit of not overwhelming new users with too many tools and stuff. Personally while developing I like to have some auto-recompile/re-run thing going on. Rust has cargo watch, each language has their own tool that is aware of project dependencies and will rebuild when one is changed, for ruby it seems the main tool is observr. I'm just starting to learn ruby and it was useful learning about a tool I'll probably use while developing

**Comment by tonywok** [comment]
I don't see the PR 😅

**Comment by bo-tato** [comment]
oops I forked it but forgot to submit PR :sweat_smile: 
https://github.com/edgecase/ruby_koans/pull/51

**Comment by tonywok** [comment]
Closed by #51
